src/transforms/metric_to_log.rs

```diff
@@ -63,6 +63,17 @@ pub struct MetricToLogConfig {
     pub metric_tag_values: MetricTagValues,
 }
 
+impl MetricToLogConfig {
+    pub fn build_transform(&self, context: &TransformContext) -> MetricToLog {
```

Comment:
Arguably the trait method should have been `build_transform`, but that's a much more invasive change.